.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://app2.fromdoppler.com
```

Comment:
Why not calling it with a more descriptive name, like _Doppler_MVC_URL_ or something like that?